src/api-server-selector.js

```diff
@@ -110,6 +110,11 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
        * Note, this does nothing when custom element is rendered.
        */
       opened: { type: Boolean },
+
+      /**
+       *
+       */
+      selectedValue: { type: String },
```

Comment:
How's this different from `value`? Where's the doc? :)

Reply:
I swear I was about to.. hence the comment block. Lol.